spec/forms/developer_search_form_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeveloperSearchForm do
+  describe '#search' do
+    before do
+      @developer = create(:developer, email: 'phanthanhhai07t1@gmail.com')
+      @language = create(:language, code: 'en')
+      @programming_language = create(:programming_language, name: 'ruby')
+      @developer_language = create(:developer_language, developer_id: @developer.id, language_id: @language.id)
+      @developer_programming_language = create(:developer_programming_language, developer_id: @developer.id,
+                                                                                programming_language_id: @programming_language.id)
+    end
+
+    context 'no select params' do
+      it 'returns all developer has create data' do
+        form = DeveloperSearchForm.new
+        expect(form.search.size).to eq(1)
+      end
+    end
+    context 'with param language' do
+      it 'correct language' do
```

Comment:
The description should make sense as an English sentence.
`correct` is an adjective.
But do you mean `collect`?
It doesn't make sense though.
Maybe you can write `returns records (or developers) that match the condition` or something.